packages/core/src/lib/push-chain/push-chain.spec.ts

```diff
@@ -822,3 +823,129 @@ describe('PushChain', () => {
     });
   });
 });
+
+describe('Deploy UEA on Push Testnet Edge Cases', () => {
```

Comment:
This should be under e2e dir
The pk should come from .env - If not found just skip the TC